index.html

```diff
@@ -35,7 +35,8 @@
 
 <h1>Latest News</h1>
 
-{% for post in site.posts limit:5 %}
+{% assign promoted_posts = site.posts | where: "promoted", true %}
```

Comment:
Liquid is supposed to support just doing this (when the attribute is truthy):

```{% assign promoted_posts = site.posts | where: "promoted" %}```

But maybe this is a new feature, or Jekyll specifically disabled it. Either way, it doesn't work. This is fine though since the more explicit version here does work.